ultralytics/engine/exporter.py

```diff
@@ -414,7 +414,9 @@ def export_onnx(self, prefix=colorstr("ONNX:")):
     @try_export
     def export_openvino(self, prefix=colorstr("OpenVINO:")):
         """YOLOv8 OpenVINO export."""
-        check_requirements("openvino>=2024.0.0")  # requires openvino: https://pypi.org/project/openvino/
+        check_requirements(
+            "openvino<=2024.0.0" if ARM64 else "openvino>=2024.0.0"
```

Comment:
@lakshanthad this line is a bit odd. Should we have two different versions here?

Reply:
@glenn-jocher OpenVINO inference for both RPi and Jetson failing with OpenVINO 2024.1.0. They only work with 2024.0.0

Reply:
@glenn-jocher Do you want to do something like:

```
suffix = "openvino"
version = "<=2024.0.0" if ARM64 else ">=2024.0.0"
check_requirements(f"openvino{suffix}{version}")
```

Reply:
@glenn-jocher Do you want to do something like:

```
version = "<=2024.0.0" if ARM64 else ">=2024.0.0"
check_requirements(f"openvino{version}")
```

Reply:
It's ok, don't worry, seems fine on review.